src/Aspire.Hosting.Yaml/Aspire.Hosting.Yaml.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can this just be shared source? Do we need to expose these types publicly?

Reply:
Not right now it doesn't have to be exposed. Could move to a yaml folder in shared, and ditch the need for a new package as well as publicly exposing it. Will do that for now until we need to